Feat(impl)/225 use caching for discovery finder requests (#693)
catenax-ng/tx-item-relationship-service: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc

Whenever a semantic model schema is requested from the Semantic Hub, it is stored locally until the cache is evicted (configurable). The IRS can preload configured schema models on startup to reduce on demand call times.

Additionally, models can be deployed with the system as a backup to the real Semantic Hub service.

=== Discovery Service

The IRS uses the Discovery Finder in order to find the correct EDC Discovery URL for the type BPN.
This URL is cached locally.

When the EDC Discovery is requested to return the EDC connector endpoint URLs for a specific BPN, the results are cached as well.

The time to live for both caches can be configured separately as described in the Administration Guide.

> Please find more information on Discovery Service see chapter "System scope and context".

Reviewer: ```suggestion
Further information on Discovery Service can be found in the chapter "System scope and context".
```

Author: Hi @ds-jhartmann , 

Ah yes, the "see" was a mistake. Completely overlooked that when I rewrote the sentence. 

However the term "Please find more information" is a correct formal term AFAIK:  https://www.linguee.de/englisch-deutsch/uebersetzung/please+find+more+information.html

To keep things short I'll just use your suggestion.
